manifests/charts/gateways/istio-ingress/values.yaml

```diff
@@ -224,6 +224,14 @@ defaults:
       # Expected values are: trace|debug|info|warning|error|critical|off
       logLevel: warning
 
+      # Additional volumes add to the gateway
```

Comment:
Can you avoid adding this to the istio-ingress and istio-egress charts? These have been deprecated in favor of the `gateway` chart. See https://github.com/istio/istio/tree/master/manifests/charts/gateway#legacy-gateway-helm-charts.

Reply:
But ```accessLogFile ``` is effective for all the gateways and sidecars，if i change the ```accessLogFile ``` to write the access log to the normal file without mounting a volume, the gateway will failed to start because of ```readOnlyRootFilesystem: true```.